app/Http/Requests/DesignDiagramRequests.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class DesignDiagramRequests extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|unique:design_diagram,name',
+            'diagram' => 'required|nullable',
+            'content' => 'required',
+        ];
+    }
+
+    public function messages()
+    {
+        return [
+            'name.required'=> __('Required'),
+            'name.unique' => __('Exists'),
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/DesignDiagramRequests.php</br>**Line**: 39</br>Each line in an array declaration must end in a comma